docs/api/ref/schemas/product_extended.yaml

```diff
@@ -0,0 +1,211 @@
+type: object
+properties:
+  additives_original_tags:
+    type: array
+    items:
+      type: string
+  additives_prev_original_tags:
+    type: array
+    items:
+      type: string
+  added_countries_tags:
+    type: array
+    items:
+      type: object
+  allergens_from_ingredients:
+    type: string
+  allergens_from_user:
+    type: string
+
+  amino_acids_prev_tags:
+    type: array
+    items:
+      type: object
+  amino_acids_tags:
+    type: array
+    items:
+      type: object
+  carbon_footprint_percent_of_known_ingredients:
+    type: integer
+  categories_properties:
+    type: object
+    properties:
+      "agribalyse_food_code:en":
+        type: string
+      "agribalyse_proxy_food_code:en":
+        type: string
+      "ciqual_food_code:en":
+        type: string
+  categories_properties_tags:
+    type: array
+    items:
+      type: string
+  category_properties:
+    type: object
+    additionalProperties:
+        description: those are properties taken from the category taxonomy
+        type: string
+  ciqual_food_name_tags:
+    type: array
+    items:
+      type: string
+  compared_to_category:
+    type: string
+    description: |
+      the category to use for comparison.
+
+      **TODO** explain how it is chosen.
+  conservation_conditions:
+    type: string
+  customer_service:
+    type: string
+    description: |
+      Contact info of customer service.
+  expiration_date:
+    type: string
+
+  link:
+    type: string
+    description: |
+      link to the product on the website of the producer
+
+  main_countries_tags:
+    type: array
+    items:
+      type: object
+
+  minerals_prev_tags:
+    type: array
+    items:
+      type: object
+  minerals_tags:
+    type: array
+    items:
+      type: object
+  owner_fields:
+    type: object
+    properties:
+      abbreviated_product_name_fr:
```

Comment:
we probably should have a pattern or something, the possible values can be names of all different fields in all languages..

Reply:
I replaced with additionalProperties, as indeed it can be a lot of fields, described in the schema.